tests/e2e/performance/csi_tests/test_bulk_pod_attachtime_performance.py

```diff
@@ -92,16 +92,16 @@ def teardown(self):
         argnames=["interface_type", "bulk_size"],
         argvalues=[
             pytest.param(
-                *[constants.CEPHBLOCKPOOL, 120],
+                *[constants.CEPHBLOCKPOOL, 4],
```

Comment:
why you changed the bulk size ?

Reply:
Oh, you are right. I've "played" with the bulk suze to check the logs and the code and forgot to change it back. Fixing it.